src/functions/interactivity/block_actions_types.ts

```diff
@@ -76,6 +76,7 @@ export type BlockActionsBody = {
     id: string;
     /**
      * @description User's handle as seen in the Slack client when e.g. at-notifying the user.
+     */
```

Comment:
There was an issue here where we weren't closing the comment, which meant that `name` wasn't part of the BlockActionsBody defined payload. This is unrelated to the core of this PR and can be removed if we want.